resources/dd-ix.net.zone

```diff
@@ -0,0 +1,60 @@
+; Check me with (abs. path required....)
+; named-checkzone dd-ix.net /.../resources/dd-ix.net.zone
+
+$TTL    1h         ; default TTL for zone
+$ORIGIN dd-ix.net. ; base domain-name
+
+@  3600  IN  SOA  ns.dd-ix.net  noc.dd-ix.net.  (
+        2020080302  ;Serial
+        7200        ;Refresh
+        3600        ;Retry
+        1209600     ;Expire
+        3600        ;Negative response caching TTL
+)
+
+;; NS Records
+@  86400   IN  NS  ans-01.ibh.de.
```

Comment:
please drop any superfluous `IN` class fields, too

(`IN` is the default class: https://bind9.readthedocs.io/en/v9.18.14/chapter3.html#textual-expression-of-rrs)